src/hooks/useProcessDetails.tsx

```diff
@@ -269,6 +277,13 @@ export function useProcessDetails(processId: number): IProcessDetails {
   ): Promise<void> => {
     try {
       let newProcess = await updateProcessStage(newStage, assignee);
+      //updateProcessStage successful, save metrics event
+      const matches = noteText.match(/<strong>(.*?)<\/strong>/g);
+      await metricsApi.submitEvent(
+        newProcess.CurrentStage,
+        newProcess.Id,
+        matches?.[0].replace(/<\/?strong>/g, "")
+      );
```

Comment:
Couple of suggestions:
1) Remove the g so we just find the first match and first capture group
2) Add 'Rework Reason:\s' to the RegEx, so that we can match that text too, and automagically remove it since it is outside the capture group.
3) Element 2 in the matches array will now just be our reason text without any of the '<strong>Rework Reason: ' and '</strong>

```suggestion
      //updateProcessStage successful, save metrics event
      const matches = noteText.match(/<strong>Rework Reason\s(.*?)<\/strong>/);
      await metricsApi.submitEvent(
        newProcess.CurrentStage,
        newProcess.Id,
        matches?.[1]
      );
```